src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -0,0 +1,99 @@
+@import "vars";
+@import "hxCheckboxFacade";
```

Comment:
Remove imports.

* "vars" is handled in `_config.scss`
* "hxCheckboxFacade" isn't even used in this file

Reply:
ok, `@import` statements removed.